### ISSUE TITLE
Screenreader can't announce table headers in sortable table

### ISSUE THREAD

**Issue by mikeslinkman** [laravel-filament/filament#13986]
### Package

filament/filament

### Package Version

v3.2.100

### Laravel Version

v11.20.0

### Livewire Version

v3.5.4

### PHP Version

8.3.9

### Problem description

When a column is marked as being sortable the header cell is rendered as a button, but in turn also gets an aria-label set based on the sorting direction. See `header-cell.blade.php`. This makes sortable tables unusable for people using a screen reader since the context for the column is lost.

### Expected behavior

At the very least the screenreader should be able to identify the column based on the label that is set. Even better would be to dynamically set the `aria-sort="sortDirection"`. However this field should only be present on the current active sortable field. See [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort) for more details.

### Steps to reproduce

The issue can be reproduced by marking a column as sortable, and using the screenreader to identify the column header. It will be read as the sorting direction.

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/filamentphp/demo

**Comment by danharrin** [comment]
Can you open a PR for this one please?

**Comment by mikeslinkman** [comment]
@danharrin Yes, I'll be able to create a PR for it, will mention you there once it is up.

**Comment by danharrin** [comment]
Thanks
